src/main/scala/apiimport/Main.scala

```diff
@@ -0,0 +1,58 @@
+package apiimport
```

Comment:
is there a naming convention for packages to be all lowercase? This looks a bit weird to me.

Reply:
I think making it `api.import` would actually make sense as it then leaves room for other api related packages